packages/amplify-e2e-tests/src/__tests__/custom_resources.test.ts

```diff
@@ -130,4 +131,23 @@ describe('adding custom resources test', () => {
       },
     });
   });
+
+  it('should use default cdk env var if set', async () => {
+    const cdkResourceName = `custom${uuid().split('-')[0]}`;
+    const destCustomResourceFilePath = path.join(projRoot, 'amplify', 'backend', 'custom', cdkResourceName, 'cdk-stack.ts');
+    const srcCustomResourceFilePath = path.join(__dirname, '..', '..', 'custom-resources', 'barebone-cdk-stack.ts');
+
+    await initJSProjectWithProfile(projRoot, {});
+    await addCDKCustomResource(projRoot, { name: cdkResourceName });
+
+    fs.copyFileSync(srcCustomResourceFilePath, destCustomResourceFilePath);
+
+    // no aws account & region set, should throw error,
+    await expect(amplifyPushAuth(projRoot)).rejects.toThrowError();
+
+    // should pickup to default cdk env vars and build cfn stack.
+    process.env.CDK_DEFAULT_ACCOUNT = 'some_id';
+    process.env.CDK_DEFAULT_REGION = process.env.CLI_REGION;
+    await amplifyPushAuth(projRoot);
```

Comment:
this lacks assertion.
we have developed the following mechanism in override tests to assert that injected values actually work
https://github.com/aws-amplify/amplify-cli/blob/d60e505391b4f8fc58c2cc4d352698a6157f1b59/packages/amplify-e2e-tests/overrides/override-auth.ts#L15-L21
Can we please use similar pattern in `barebone-cdk-stack.ts` ?